### PR TITLE
launch-blogpost

### PR DESCRIPTION
**Pull request by nandhinidevie**
work in progress

### CODE REVIEW

File: pages/blogs/launch.ftd [hunk 1]
```diff
@@ -0,0 +1,211 @@
+-- import: ui.fifthtry.com/components/page as p
```
Comment (amitu) [inline pages/blogs/launch.ftd:1]: @nandhinidevie we have to put the blog UI in this package. The blog content has to be in ft repos' frontend folder.

**Comment by nandhinidevie** [comment]
@amitu can you merge this PR?

File: index.ftd [hunk 2]
```diff
@@ -20,6 +20,8 @@ spacing: $ds.spaces.vertical-gap.medium.gap
 	- [Privacy page](/scenarios/about/privacy-page/)
 	- [Team page](/scenarios/about/team-page/)
 	- [About Us page](/scenarios/about/)
+	- [Blog](/scenarios/blogs/)
+	- [Blog Homepage](/scenarios/blogs/blog-home/)
```
Comment (amitu) [inline index.ftd:24]: /blogs/ is not blog home page? Where is blog post page?

File: pages/blogs/home.ftd [hunk 3]
```diff
@@ -0,0 +1,96 @@
+-- import: ui.fifthtry.com/components/page as p
```
Comment (amitu) [inline pages/blogs/home.ftd:1]: These files can only contain structure/components, not content. Move the content to scenario files.

**Pull request merged**